### PR TITLE
connection timeout configuration

### PR DESCRIPTION
**Pull request by henriquechehad**
Issue: https://github.com/nuveo/prest/issues/126

### CODE REVIEW

File: adapters/postgres/postgres_test.go [hunk 1]
```diff
@@ -791,3 +791,24 @@ func TestColumnsByRequest(t *testing.T) {
 		}
 	}
 }
+
+func TestTimeout(t *testing.T) {
```
Comment (felipeweb) [inline adapters/postgres/postgres_test.go]: This test is a connection test.. the correct file to write this test is the `conn_test.go` in connection package

**Comment by henriquechehad** [comment]
WIP

**Comment by henriquechehad** [comment]
@felipeweb it not worked because uses `postgres.Query` in the test and returns error `import cycle not allowed in test`. suggestions?

**Comment by felipeweb** [comment]
@henriquechehad could mock this queries, using sqlmock no?

**Comment by henriquechehad** [comment]
@felipeweb i think it must use `postgres.Query` func to test itself with the return

**Comment by felipeweb** [comment]
@henriquechehad  ok so move the test to postgres package

**Pull request merged**